README.md

```diff
@@ -319,7 +320,9 @@ where pins cannot be placed. Can be used multiple times.
 random.
 - ``-group_pins``. Specify a list of pins to be placed together on the die boundary.
 - ``-corner_avoidance distance``. Specify the distance (in micron) from each corner to avoid placing pins.
-- ``-min_distance distance``. Specify the minimum distance (in micron) between pins in the die boundary.
+- ``-min_distance distance``. Specify the minimum distance between pins in the die boundary.
+It can be in microns (default) or in number of tracks between each pin.
+- ``-min_distance_in_tracks``. Flag that allows set the min distance in number of tracks instead of microns.
```

Comment:
This seems confusing to have an option to change the units of a different option.  Are we trying to support both or is one depracated?

Reply:
The goal is to support both -- i.e. to switch the min_distance value to either specify in multiples of routing tracks or in microns. Today, I believe the default is value is 2 tracks. However if we need to change that to one track, we will need to determine the value from the tech file and specify that as a micron value

editPin in Innovus has a similar functionality too, where in the GUI there is a pull down to specify pin spacing either in track pitch or um

Reply:
The editPin has -unit {MICRON | TRACK} which seems similar.